docs/book/v5/upgrading/UPGRADE-5.3.md

```diff
@@ -11,27 +11,34 @@ This document only covers upgrading from version 5.2.
 
 -------------------------
 
-* [Update Phpstan memory limit](#update-phpstan-memory-limit)
+* [Update PHPStan memory limit](#update-phpstan-memory-limit)
 * [Update anonymization](#update-anonymization)
 * [Update User status and remove isDeleted properties](#update-user-status-and-remove-isdeleted-properties)
 * [Update dotkernel/dot-mail to version 5.0](#update-dotkerneldot-mail-to-version-50)
 * [Add post install script](#add-post-install-script)
 * [Remove post-create-project-cmd](#remove-post-create-project-cmd)
+* [Ignore development files on production env](#ignore-development-files-on-production-env)
 * [Update security.txt](#update-securitytxt)
 * [Update coding standards](#update-coding-standards)
-* [Fix codecov.yml](#fix-codecovyml)
 * [Update Qodana configuration](#update-qodana-configuration)
 * [Remove laminas/laminas-http](#remove-laminaslaminas-http)
 
-### Update Phpstan memory limit
+### Update PHPStan memory limit
+
+Following PHPStan's introduction in version 5.2 for the reasons described on the [Dotkernel blog](https://www.dotkernel.com/php-development/static-analysis-replacing-psalm-with-phpstan/) a minor issue has cropped up:
+ with the default `memory_limit=128M` on our WSL containers, PHPStan runs out of memory
 
 * Add the `--memory-limit 1G` option to the `static-analysis` script found in `composer.json`
-  > With the default `memory_limit=128M` on our WSL containers, PHPStan runs out of memory
-  >
   > Note that you can set the memory limit to a value of your choosing, with a recommended minimum of 256M
 
 ### Update anonymization
 
+By default, Dotkernel API uses "soft delete" for it's `User` entities in order to preserve the database entries.
+
```

Comment:
do we need this much spacing between these phrases? the other sections also have 3 separate paragraphs, but i think it's better to merge them as they are on a single topic